README.md

```diff
@@ -62,12 +76,21 @@ flutter run
 ```
 
 ## Linux
-If you want to build Jellyflix for Linux you need some additional dependencies: `libmpv-dev, mpv, libsecret-1-dev, libjsoncpp-dev
+
+If you want to build Jellyflix for Linux you need some additional dependencies:
+
+```
+sudo apt install build-essential cmake clang libgtk-3-dev libjsoncpp-dev libmpv-dev libsecret-1-dev mpv
```

Comment:
The dependencies you added are already required for "normal" Flutter development. If you follow the the get-started part in the Flutter documentation you should already have them installed. This part of the documentation only states the additional dependencies you need for Jellyflix development. 
We could differentiate between building for Linux and just running on Linux.

Reply:
I think my needs could be satisfied with something as minimal as an updated link, or lik a "see these instructions at the bottom of the page" type thing.  this issue was motivated by how difficult it felt to get up and running, but, maybe i was just being a baby.  i will revisit the steps again later and see if I feel the same way.